### PR TITLE
UI base

### PR DESCRIPTION
**Pull request by JustAnOtherCoder13**
start constructing theme. add colors and texAppearence.
start constructing UI. construct fragment_property_detail with custom view

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -7,6 +7,13 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
 
+    <supports-screens
```
Comment (fodeguirassy) [inline app/src/main/AndroidManifest.xml:10]: resizable is `true` by default

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationCustomView.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    public AddPropertyInformationCustomView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+    }
+
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+
+        View addPropertyInformation = LayoutInflater.from(getContext()).inflate(R.layout.custom_view_add_property_information,this,false);
```
Comment (fodeguirassy) [inline app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationCustomView.java]: `addView()` is not necessary. 
See [https://stackoverflow.com/questions/4328838/create-a-custom-view-by-inflating-a-layout](url)

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationTypeCustomView.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    public AddPropertyInformationTypeCustomView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+    }
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+
+        View addPropertyInformationType = LayoutInflater.from(getContext()).inflate(R.layout.custom_view_add_property_information_type,this,false);
```
Comment (fodeguirassy) [inline app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationTypeCustomView.java]: Unnecessary `addView()`

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertySoldCustomView.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    public AddPropertySoldCustomView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+    }
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+
+        View addPropertySold = LayoutInflater.from(getContext()).inflate(R.layout.custom_view_add_property_sold,this,false);
```
Comment (fodeguirassy) [inline app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertySoldCustomView.java]: Same here

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailDescriptionCustomView.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+    public DetailDescriptionCustomView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+    }
+
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+
+        View detailDescription = LayoutInflater.from(getContext()).inflate(R.layout.custom_view_detail_description,this,false);
```
Comment (fodeguirassy) [inline app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailDescriptionCustomView.java]: Same

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailInformationCustomView.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    public DetailInformationCustomView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+    }
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+
+        View detailInformation = LayoutInflater.from(getContext()).inflate(R.layout.custom_view_detail_information,this,false);
```
Comment (fodeguirassy) [inline app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailInformationCustomView.java]: Same here

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailMediaCustomView.java [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+    public DetailMediaCustomView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+    }
+
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+
+        View detailMedia = LayoutInflater.from(getContext()).inflate(R.layout.custom_view_detail_media,this,false);
```
Comment (fodeguirassy) [inline app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailMediaCustomView.java]: Same here

File: app/src/main/res/layout/custom_view_add_property_information_type.xml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+    <TextView
+        android:id="@+id/add_property_information_type_custom_view_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toTopOf="@id/add_property_information_type_custom_view_spinner"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:text="@string/property_type"
+        android:textAppearance="@style/TextAppearance.RealEstateManagerStyles.Subtitle1"
```
Comment (fodeguirassy) [inline app/src/main/res/layout/custom_view_add_property_information_type.xml]: format

File: app/src/main/res/layout/custom_view_add_property_information_type.xml [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+        />
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/add_property_information_type_custom_view_spinner"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="@id/add_property_information_type_custom_view_title"
+        app:layout_constraintEnd_toEndOf="@id/add_property_information_type_custom_view_title"
+        app:layout_constraintTop_toBottomOf="@id/add_property_information_type_custom_view_title"
```
Comment (fodeguirassy) [inline app/src/main/res/layout/custom_view_add_property_information_type.xml]: format

File: app/src/main/res/layout/custom_view_add_property_sold.xml [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+    <TextView
+        android:id="@+id/add_property_sold_custom_view_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/sold"
+        android:layout_marginStart="@dimen/basic_marge"
+        android:textAppearance="@style/TextAppearance.RealEstateManagerStyles.Headline4"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
```
Comment (fodeguirassy) [inline app/src/main/res/layout/custom_view_add_property_sold.xml]: format

**Review by fodeguirassy — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (JustAnOtherCoder13) [reply app/src/main/AndroidManifest.xml:10]: done

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationCustomView.java (hunk 2 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationCustomView.java]: done

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationTypeCustomView.java (hunk 3 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationTypeCustomView.java]: ok

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertySoldCustomView.java (hunk 4 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertySoldCustomView.java]: ok

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailDescriptionCustomView.java (hunk 5 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailDescriptionCustomView.java]: ok

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailInformationCustomView.java (hunk 6 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailInformationCustomView.java]: ok

File: app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailMediaCustomView.java (hunk 7 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/DetailMediaCustomView.java]: ok

File: app/src/main/res/layout/custom_view_add_property_information_type.xml (hunk 8 above)
Comment (JustAnOtherCoder13) [reply app/src/main/res/layout/custom_view_add_property_information_type.xml]: done

File: app/src/main/res/layout/custom_view_add_property_information_type.xml (hunk 9 above)
Comment (JustAnOtherCoder13) [reply app/src/main/res/layout/custom_view_add_property_information_type.xml]: done

File: app/src/main/res/layout/custom_view_add_property_sold.xml (hunk 10 above)
Comment (JustAnOtherCoder13) [reply app/src/main/res/layout/custom_view_add_property_sold.xml]: done

**Pull request merged**